### DISCUSSION TITLE
Easier way to pause screen in the editor mode

### DISCUSSION THREAD

**Discussion by JaylyDev** [Mojang/minecraft-editor#85] [Feedback]
Currently we can pause the game via `Ctrl + Q`. But this combination of keybind is not widely use, espically in Minecraft. Please add a button either:
- In the editor gui (ie. A button on top of the bar with lots of empty spaces), so we can hover the mouse and click one button to go to the pause screen.
![image](https://user-images.githubusercontent.com/65847850/225334363-62cd9318-c019-4f12-b2e9-670bac3e31b2.png)

- Or allow us to go to the pause screen via `Esc` key, as that's what we all use to pause the game.

This is because people who are new to use this editor have some trouble going to the pause screen without reading documentation, or thinking that my escape key doesn't work, this scenario happens to me too when I first started using editor. I don't like having to press multiple keys or pause screen from "File" tab.

Also I think [this YouTube video](https://youtu.be/uCoEIUpEJ-o?t=160) demostrate this issue.

**Comment by minecraftchloe** [comment]
Agreed! We have planned to eliminate the pause screen as you see it in the flow today, and instead port over those elements (Settings, Friends) to the top menu bar -- File dropdown.
